types/adeira__js/index.d.ts

```diff
@@ -0,0 +1,25 @@
+// Type definitions for @adeira/js 1.2
+// Project: https://github.com/adeira/universe/tree/master/src/js
+// Definitions by: Martin Zlámal <https://github.com/me>
```

Comment:
<!--
  1 failure:  This line should ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/adeira__js/index.d.ts;
  Line: 3;
-->
- :no_entry_sign: This line should have your github username in it, not /me